lib/Core/Executor.h

```diff
@@ -95,9 +97,29 @@ class Executor : public Interpreter {
   bool allBlockCollected;
   std::set<llvm::BasicBlock *> visitedBlocks;
   float blockCoverage;
-  std::string covInterestSourceFileName;
+  std::string interestedSourceFileName;
```

Comment:
Why are we changing this file name? The previous name seems to be more meaningful.